### PR TITLE
[Spec] add the restricted elements

### PR DESCRIPTION
**Pull request by helloahn**
The elements for audio & edgeAI have been added.
To make it clear, the additional macros have been added.

The generated meson build script

After
```
meson 
...
'-Drestricted-elements=capsfilter input-selector output-selector queue tee valve appsink appsrc audioconvert \
audiorate audioresample audiomixer videoconvert videocrop videorate videoscale videoflip videomixer \
compositor fakesrc fakesink filesrc filesink audiotestsrc videotestsrc jpegparse jpegenc jpegdec pngenc \
pngdec tcpclientsink tcpclientsrc tcpserversink tcpserversrc udpsink udpsrc xvimagesink ximagesink \
evasimagesink evaspixmapsink glimagesink theoraenc lame vorbisenc wavenc volume oggmux avimux \
matroskamux v4l2src avsysvideosrc camerasrc tvcamerasrc pulsesrc fimcconvert tizenwlsink gdppay \
gdpdepay join audioamplify audiochebband audiocheblimit audiodynamic audioecho audiofirfilter \
audioiirfilter audioinvert audiokaraoke audiopanorama audiowsincband audiowsinclimit scaletempo \
stereo rtpdec rtspsrc rtspclientsink zmqsrc zmqsink mqttsrc mqttsink udpsrc udpsink multiudpsink \
queryclient queryserversrc queryserversink ' \
...
build
```

Before
```
meson 
...
'-Drestricted-elements=capsfilter input-selector output-selector queue tee valve appsink appsrc audioconvert \
audiorate audioresample audiomixer videoconvert videocrop videorate videoscale videoflip videomixer \
compositor fakesrc fakesink filesrc filesink audiotestsrc videotestsrc jpegparse jpegenc jpegdec pngenc \
pngdec tcpclientsink tcpclientsrc tcpserversink tcpserversrc udpsink udpsrc xvimagesink ximagesink \
evasimagesink evaspixmapsink glimagesink theoraenc lame vorbisenc wavenc volume oggmux avimux \
matroskamux v4l2src avsysvideosrc camerasrc tvcamerasrc pulsesrc fimcconvert tizenwlsink gdppay \
gdpdepay join'
...
build
```

See also: #3244

Signed-off-by: Hyoung Joo Ahn <hello.ahn@samsung.com>

**Self evaluation:**
1. Build test: [*]Passed [ ]Failed [ ]Skipped
2. Run test: [ ]Passed [ ]Failed [*]Skipped

### CODE REVIEW

**Review by myungjoo — APPROVED** [review]
LGTM

**Review by anyj0527 — APPROVED** [review]
LGTM ✔

File: packaging/nnstreamer.spec [hunk 1]
```diff
@@ -565,7 +565,10 @@ Provides additional gstreamer plugins for nnstreamer pipelines
...
+%define restricted_element_base     'capsfilter input-selector output-selector queue tee valve appsink appsrc audioconvert audiorate audioresample audiomixer videoconvert videocrop videorate videoscale videoflip videomixer compositor fakesrc fakesink filesrc filesink audiotestsrc videotestsrc jpegparse jpegenc jpegdec pngenc pngdec tcpclientsink tcpclientsrc tcpserversink tcpserversrc udpsink udpsrc xvimagesink ximagesink evasimagesink evaspixmapsink glimagesink theoraenc lame vorbisenc wavenc volume oggmux avimux matroskamux v4l2src avsysvideosrc camerasrc tvcamerasrc pulsesrc fimcconvert tizenwlsink gdppay gdpdepay join '
+%define restricted_element_edgeai   'rtpdec rtspsrc rtspclientsink zmqsrc zmqsink mqttsrc mqttsink udpsrc udpsink multiudpsink queryclient queryserversrc queryserversink '
```
Comment (gichan-jang) [inline packaging/nnstreamer.spec:569]: `udpsink` is duplicated.

Comment (helloahn) [reply packaging/nnstreamer.spec:569]: nice catch :+1:  it's fixed.

**Review by gichan-jang — APPROVED** [review]
LGTM

**Review by again4you — APPROVED** [review]
LGTM!

**Pull request merged**